docs/CHANGELOG.md

```diff
@@ -12,6 +12,15 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
 <!-- PUT THE NEW CHANGELOG ENTRY RIGHT BELOW THIS -->
 <!-------------------------------------------------->
 
+## PLACEHOLDER FOR RELEASE DATE
+
+* The data for all samples, other than multiplexed samples, can now be downloaded as either a `SingleCellExperiment(R)` or `AnnData(Python)` object contained in a RDS or HDF5 file, respectively.
+* The sample metadata found in `single_cell_metadata.tsv` has been updated to include ontology term ids for age, sex, organism, ethnicity, diagnosis, and tissue location, when available.
+* All samples now have an assigned `participant_id`, which can be found in `single_cell_metadata.tsv`.
```

Comment:
I think it'd be helpful to give more information about what we _used to do_ to provide more context for the potential impact:
```suggestion
* All samples now have an assigned `participant_id`, which can be found in `single_cell_metadata.tsv`.
Previously, `participant_id` were only assigned when multiple samples mapped to the same participant for most projects.
```